.github/workflows/regression.yml

```diff
@@ -54,6 +54,18 @@ jobs:
       shell: bash
       run: sudo swapoff -a
 
+
+    # needed because clickhouse for some reason produces an error the first
+    # time a benchmark is run. The next benchmark run will work and overwrite the
```

Comment:
Let's check if it remains after my PR will be merged. I think it will be not necessary(or I will check what happens)